port/zephyr/libcoap/tls.c

```diff
@@ -21,6 +24,12 @@ static sec_tag_t libcoap_sec_tag = {
         515765868,
 };
 
+/* Use mbedTLS macros which are IANA ciphersuite names prepended with MBEDTLS_ */
+#define GOLIOTH_CIPHERSUITE_ENTRY(x) _CONCAT(MBEDTLS_, x)
+
+static int golioth_ciphersuites[] = {
+    FOR_EACH_NONEMPTY_TERM(GOLIOTH_CIPHERSUITE_ENTRY, (, ), GOLIOTH_CIPHERSUITES)};
```

Comment:
NIT: Can we put the `}` on a newline or is clang-format complaining?

Reply:
I'll check, though it was clang-format that formatted this code in the first place.

Reply:
`clang-format` didn't complain after putting `}` on a new line.